cfpb/retirement
# Ingles

Minor fixes for Spanish translation and ssa_check

## Changes

- fixes reference to 'inglés' on Spanish About page

## Additions

- Adds a `--server` option to the `check_ssa` command to help automated testing

## Testing

- check `/retirement/before-you-claim/about/es/`and the graf containing `inglés` should look like this, which Hector says is right:

<img width="663" alt="inglesia" src="https://cloud.githubusercontent.com/assets/515885/15507997/c6bf6614-219b-11e6-8b5d-4357f05217a1.png">

- run `./manage.py check_ssa --server prod` to run our API test against production. It should return a log entry similar to this:  
  ```
  OK,2016-05-24 09:54,prod,200,,,5.9
  ```

## Review

- @amymok @niqjohnson @marteki 

## Todos

- I'm hoping to tag for deployment after this PR

## Checklist
